Jenkinsfile

```diff
@@ -15,6 +15,13 @@ pipeline {
       }
     }
 
+    stage('Test') {
+      steps {
+        sh './test/test.sh postgres'
```

Comment:
this might be my own ignorance of jenkinsfiles, but why is the above not:
```
./test/test.sh 'postgres'
```

Reply:
I made no assumptions... just being consistent with the other Jenkinsfiles